plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1194,6 +1194,7 @@ blockly.block.string=String of type %2 parameter %1 %3 %4
 blockly.block.string.tooltip="quotable text" allows to have a quoted sentence.
 blockly.block.entity=Entity parameter %1 for %2 %3 %4
 blockly.block.item=Item parameter %1 %2 %3
+blockly.block.is_mod_loaded=Is the mod %1 loaded
```

Comment:
I'd change this text to "Is the mod of ID %1 loaded" to be clearer, other than that PR looks good to me